sdcm/nemesis.py

```diff
@@ -3387,7 +3387,18 @@ def decommission_post_action():
             retry=0,
         )
 
-        log_follower = self.target_node.follow_system_log(patterns=["DECOMMISSIONING: unbootstrap starts"])
+        terminate_patterns = ["DECOMMISSIONING: unbootstrap starts",
+                              "DECOMMISSIONING: unbootstrap done",
+                              "becoming a group 0 non-voter",
+                              "became a group 0 non-voter",
+                              "leaving token ring",
+                              "left token ring",
+                              "Finished token ring movement"]
+
+        terminate_pattern = random.choice(terminate_patterns)
```

Comment:
Please randomize based on the nemesis seed... (or some seed)

So it would be reproducable

Reply:
It won't be reproducible anyway, it's so racy that it's very very hard to reproduce it.
But we can try I guess...

Reply:
yep, it is difficult to catch exactly the moment after log detected. But various line and dataset per job could help with that.